src/crypto/subtle.cr

```diff
@@ -23,7 +23,7 @@ module Crypto::Subtle
     constant_time_byte_eq(v, 0) == 1
   end
 
-  def self.constant_time_byte_eq(x, y)
+  def self.constant_time_byte_eq(x : UInt8 | Int32, y : Int32 | UInt8) : UInt8 | Int32
```

Comment:
This doesn't look reasonable. This probably needs to be one of those `forall T`.

Reply:
As the name suggests, this method implements byte equality. Its operands must be `UInt8`.

`Int32` is a contamination from a sloppy unit test setup.